spec/spec.md

```diff
@@ -139,6 +139,19 @@ These features are deprecated, but can be supported by wallet endpoints to maint
 
 - [`solana:signTransactions`](#sign_transactions)
 
+## Account Profiles
+
+Account profiles are labels that are used to decribe an authorized account's capabilities and intended use. Each authorized account that is returned in [`authorize`](#authorize) request response can include a list of these profiles. The dapp endpoint can use these labels in combination with the account's list of [`features`](#feature-identifiers) to determine how each acount should be used. 
+
+- `primary`
+  - The users primary account. This account will be used for sign in, and is expected to have the users primary assets. 
+- `feePayer`
+  - Fee payer accounts, when present, should be used to pay for transaction fees. This profile is incompatible with the `immutable` profile, a fee payer account cannot be `immutable`. 
+- `mutable`
```

Comment:
Do we need both `mutable` and `immutable`, or does the absence of one imply the other?

Reply:
I could go either way, I am not certain what is best. both have drawbacks I think

A) With both `mutable` and `immutable`, its more explicit. tho it also adds the incompatibilities (`mutable` and `feepayer` cannot also be `immutable`)

B) With only `mutable`, the absence would imply immutability. `feePayer` would imply `mutable` as well which makes sense. `primary` would not imply mutable, it would need to be specified. 

C) With only `immutable`, the absence would imply mutability. `feePayer` would be incompatible with `immutable` as it would not make sense to have an immutable fee payer. 

I am leaning towards option B here, only having `mutable`. `primary` (and `identity`) accounts imply immutability if not specified. `feePayer` and `mutable` would be allowed but redundant.

Reply:
I can think of one benefit for option C - if a wallet labels nothing. The default (and backward-compatible) behavior for an unlabeled account would be mutable.

Reply:
yeah thats a good point. gonna mull on this a bit more.